app/presenters/publishing_api/organisation_presenter.rb

```diff
@@ -495,5 +497,24 @@ def high_profile_groups_links
     def roles_links
       item.roles.distinct.pluck(:content_id)
     end
+
+    def default_news_image
+      return unless item.default_news_image
+      return { url: default_news_image_url } if svg?
+
+      {
+        url: default_news_image_url(:s300),
+        high_resolution_url: default_news_image_url(:s960),
+      }
+    end
+
+    def default_news_image_url(size = nil)
+      size ? item.default_news_image.file.url(size) : item.default_news_image.file.url
+    end
+
+    def svg?
```

Comment:
Minor, but I'd suggest this method name should either mention default news image or take in the image as an argument - just that it doesn't make too much sense to ask the presenter as a whole if it is an svg.